src/components/PageHeader/_page-header.scss

```diff
@@ -1,18 +1,10 @@
 .title-banner__container {
-  // padding: rem(var(--text-base-size-px) * 2) 0;
-  padding: rem(32) 0;
+  padding-left: var(--space-xl);
```

Comment:
--space-xl will change size depending on the breakpoint (based on spacings in Wellcome Design System - https://zpl.io/bJWJNgW)

is that what you need?

if these values are static or differ in any way then use multiples of --space-unit

Reply:
Yep, --space-xl is intended.

Here's the settings for it:

```css
:root {
  // 32px
  --space-xl: calc(4 * var(--space-unit));

  @include mq(sm) {
    // 48px
    --space-xl: calc(6 * var(--space-unit));
  }

  @include mq(md) {
    // 64px
    --space-xl: calc(8 * var(--space-unit));
  }
}

Reply:
👍 great, just checking